### PR TITLE
Outlining structure for 7 to latest migration guide

### PR DESCRIPTION
**Pull request by sofietoft**
This is the first draft for adding a migration guide from Umbraco 7 to the latest CMS version.

**UPDATE**
Instead of writing a step-by-step guide, I am moving all the version-specific upgrade docs onto the new platform.
This includes Cloud and CMS.

* [x] Clean up Version Specific upgrade article for CMS
* [x] Format "Migrate content to Umbraco 8" article for new platform
* [x] Format "Upgrade to Umbraco 7" article for new platform
* [x] Format "Migrate 7 to 8 on Cloud" article for new platform
* [x] Clean up Umbraco Forms version specific upgrades document
* [x] Figure our how to handle "7 to 8 Data Types migration" article on new platform
* [x] Put Breaking Changes into expandables in a new "Breaking Changes" article

### CODE REVIEW

File: umbraco-cloud/upgrades/migrate-from-umbraco-7-to-8.md [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+---
+
+# Migrate from Umbraco 7 to Umbraco 8 on Umbraco Cloud
+
+{% hint style="warning" %}
+Due to [a known issue](https://github.com/umbraco/Umbraco-CMS/issues/7914), it is **not possible to directly migrate your Umbraco 7 project to the latest Umbraco 8**.
+
+As a workaround, you can either
+
+* Migrate to **version 8.5** as a first step, and then, post-migration, carry out a normal Umbraco upgrade to the latest version of Umbraco 8, or
```
Comment (jonat123) [inline umbraco-cloud/upgrades/migrate-from-umbraco-7-to-8.md:12]: Is this workaround even possible on Umbraco Cloud? How would one go about getting a Umbraco 8.5 project? 

Maybe I am missing something? 🤔

File: umbraco-cloud/upgrades/migrate-from-umbraco-7-to-8.md [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+<iframe width="800" height="450" title="Migrate an Umbraco Cloud project from 7 to 8: Introduction" src="https://www.youtube.com/embed/wNIDdgdAt8s?rel=0" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
+
+You can find the full playlist here: [Migrate an Umbraco Cloud project from 7 to 8](https://www.youtube.com/playlist?list=PLgX62vUaGZsGceCXveMkprlDV5a3K94db)
+
+## Prerequisites
+
+* A Umbraco 7 Cloud project running **the latest version of Umbraco 7**
+* Make sure Umbraco Forms data is not handled as content
+  * See [Umbraco Forms on Cloud](../deployment/umbraco-forms-on-cloud.md#how-forms-are-handled-on-umbraco-cloud) for more details on how to check the setting
+* A clean Cloud project running the latest version of Umbraco 8 with **at least 2 environments**
```
Comment (jonat123) [inline umbraco-cloud/upgrades/migrate-from-umbraco-7-to-8.md:31]: Again this is related to the above comment, the top warning says that it is not possible to upgrade to the latest version of Umbraco 8, while the prerequisite is that the v8 project needs to be the latest v8 version, which seems contradictory 🤔

File: umbraco-cloud/upgrades/migrate-from-umbraco-7-to-8.md (hunk 1 above)
Comment (sofietoft) [reply umbraco-cloud/upgrades/migrate-from-umbraco-7-to-8.md:12]: Yeah, that's a good observation 🤔 
It should still be possible, you would need to do the migration from 7 to 8.5 locally, then migrate it into a Cloud project running 8.latest afterward.

File: umbraco-cloud/upgrades/migrate-from-umbraco-7-to-8.md (hunk 2 above)
Comment (sofietoft) [reply umbraco-cloud/upgrades/migrate-from-umbraco-7-to-8.md:31]: Following the comment above - the first prerequisites would instead be "A Umbraco 7 project upgraded to Umbraco 8.5 locally".

Then we could add a link to the (on-prem) migration docs before this migration starts.

File: umbraco-cloud/upgrades/migrate-from-umbraco-7-to-8.md (hunk 1 above)
Comment (jonat123) [reply umbraco-cloud/upgrades/migrate-from-umbraco-7-to-8.md:12]: That makes sense 👍

File: umbraco-cloud/upgrades/migrate-from-umbraco-7-to-8.md (hunk 2 above)
Comment (jonat123) [reply umbraco-cloud/upgrades/migrate-from-umbraco-7-to-8.md:31]: Yeah i agree, it makes sense to change the prerequisite :)

**Pull request closed without merging**